app/views/editorial/nodes/show.html.haml

```diff
@@ -47,6 +47,11 @@
     %tr
       %td State
       %td= node.state
+    -# TODO: refactor this to properly handle different options per Node subclass
+    - if node.options.respond_to?(:toc)
```

Comment:
What I really want to do is iterate through all options for the given type of node. But it will be easier to figure out how to do that once we have a second option.